File: market/python/TimePeriodVolumeRanking.py
import requests
import json

# set the url
url = "https://api.deversifi.com/v1/pub/tokenRanking/{currency}"

# set the parameters
url = url.format(currency="ETH")

# give start- and enddates as params
params = {
    "startDate": "1577836800000",
    "endDate": "1577923200000"
}

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers, params=params).text)

# print the response from dictionary
print(json.dumps(response, indent=2))

File: market/js/TimePeriodVolumeRanking.js
var request = require("request");
// set parameters
const currency = "ETH";
const startDate = "startDate=1577836800000";
const endDate = "endDate=1577923200000";
// set url
const url = "https://api.rhino.fi/v1/pub/tokenRanking/" + currency + "?" + startDate + "&" + endDate;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});